lib/srv/forward/sshserver.go

```diff
@@ -412,6 +412,11 @@ func (s *Server) GetClock() clockwork.Clock {
 	return s.clock
 }
 
+// GetUtmpPath returns the optional override of the utmp and wtmp path.
```

Comment:
```
// GetUtmpPath returns returns the optional override of the utmp and wtmp path.
// These values are never set for the forwarding server because utmp and wtmp
// are updated by the target server and not the forwarding server.
```